Formula/spdlog.rb

```diff
@@ -0,0 +1,37 @@
+class Spdlog < Formula
+  desc "Super fast C++ logging library."
+  homepage "https://github.com/gabime/spdlog"
+  url "https://github.com/gabime/spdlog/archive/v0.11.0.tar.gz"
+  sha256 "8c0f1810fb6b7d23fef70c2ea8b6fa6768ac8d18d6e0de39be1f48865e22916e"
+  head "https://github.com/gabime/spdlog.git", :branch => "master"
```

Comment:
Much appreciated. That's my only nit on this formula, the rest LGTM to me. We'll leave it open for a bit to give other maintainers an opportunity to provide feedback.